docs/main/Solutioning/openebs-on-kubernetes-platforms/gke.md

```diff
@@ -313,4 +313,5 @@ root@nginx-deployment-7bf66b59f5-mxcdp:/volume#
 ## See Also
 
 - [Replicated PV Mayastor Installation on MicroK8s](microkubernetes.md)
-- [Replicated PV Mayastor Installation on Talos](talos.md)
\ No newline at end of file
+- [Replicated PV Mayastor Installation on Talos](talos.md)
+- [Provisioning Read-Write-Many (RWX) PVCs](rwx.md)
```

Comment:
I think this is not relevant under this folder/group. It needs a different group which can include
- rwx
- object store (in the future, when we solution with minio)

Reply:
@avishnu Can we create a folder named "Operations" to place the RWX docs?

Reply:
I think, Read Write Many will be better for the folder name. For the current topic, we could call "Provisioning NFS PVCs"